defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
+   "--text",
+   "-n",
+   "--no-self-upgrade",
+   "-m {{ letsencrypt_email }}",
+   "--domains {{ _letsencrypt_domains | default(letsencrypt_domain) }}",
+   "--agree-tos",
+   "--standalone",
+   "--expand"]
```

Comment:
Will update the default params to another one... my issue is that having them together is better...for me at least -- I copy/paste the default params and change what I need to and delete what I don't -- so seeing what's actually set and changing things is easiest for me...

One way we can make this easier: 

- an option to run in standalone mode
- an option to generate the apache config 
- an option to generate diffie hellman parameters with 4096 bits (security is important to me)